_episodes/05-comparing-agile-methodologies.md

```diff
@@ -48,38 +48,45 @@ They both provide a set of practical tools that allow you to work according to t
 > ## Exercise: key differences between Scrum and Kanban
 > See the following uncomplete table:
 >
-> | Aspect              | Scrum | Kanban |
-> |---------------------|-------|--------|
-> | Cadance             | Regular fixed length sprints (1-4 weeks) | Continuous flow |
-> | Release Methodology | A     | B      |
-> | Roles               | C     | D      |
-> | Key Metrics         | E     | F      |
-> | Change Philosophy   | G     | H      |
+> | Aspect                        | Scrum | Kanban |
+> |-------------------------------|-------|--------|
+> | Release Methodology           | At the end of each sprint | Continuous delivery |
+> | Cadance                       |      |       |
+> | Roles                         |      |       |
+> | Amount of work estimation     |      |       |
+> | Change Philosophy             |      |       |
+> | Meetings                      |      |       |
 >
-> It denotes the differences between Scrum and Kanban for 5 key aspects.
-> **The first row (Cadance) is filled, but your task is to fill the last 4 rows of the table (for cells A-H).**
+> It denotes the differences between Scrum and Kanban for 6 key aspects.
+> **The first row (Release Methodology) is filled.
+> Your task is to fill the last 5 rows of the table.**
```

Comment:
Would it be a good idea to leave the letters A-H in the incomplete table so the answer to this exercise is a simple list of letter to number combinations (e.g. C-1)?

Reply:
A little description of how the answer should look like should then be added to this piece of text. We could even add letters for the first row and add the two answers to the list below and then give the first two answers as an example instead of showing them in the table above.

Reply:
I do think, it is good to have the complete filled in table in the solutions, but we could also add the letter-number combinations, so it is easy to check your answers.